app/api/routers/breeze_buddy/telephony/callbacks/handlers.py

```
@@ -197,59 +299,207 @@ async def handle_callback_status(request: Request, provider: str) -> Response:
+        # Valid selection - start recording and return Stream XML
+        logger.info(
+            f"[PlivoIVR] Valid selection: {selected_template['name']} "
+            f"(template_id: {selected_template['id']})"
+        )
+
+        # Extract CallUUID from form data for recording
+        call_uuid = str(form.get("CallUUID", ""))
+        if call_uuid:
+            try:
+                start_call_recording(call_uuid)
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)